client/login/wp-login/index.jsx

```diff
@@ -65,14 +65,13 @@ export class Login extends Component {
 		this.recordPageView( this.props );
 	}
 
-	// @TODO: Please update https://github.com/Automattic/wp-calypso/issues/58453 if you are refactoring away from UNSAFE_* lifecycle methods!
-	UNSAFE_componentWillReceiveProps( nextProps ) {
-		if ( this.props.twoFactorAuthType !== nextProps.twoFactorAuthType ) {
-			this.recordPageView( nextProps );
+	componentDidUpdate( prevProps ) {
+		if ( this.props.twoFactorAuthType !== prevProps.twoFactorAuthType ) {
+			this.recordPageView( this.props );
 		}
 
-		if ( this.props.socialConnect !== nextProps.socialConnect ) {
-			this.recordPageView( nextProps );
+		if ( this.props.socialConnect !== prevProps.socialConnect ) {
+			this.recordPageView( this.props );
```

Comment:
`this.recordPageView` now always uses `this.props`. Mind removing the argument and using `this.props` directly in the method?

Reply:
True! Fixed via https://github.com/Automattic/wp-calypso/pull/61165/commits/6b288dc4e2fea993abaaf24bdf638d1d3aba78c0